tests/extras/test_annotate_vcf.py

```diff
@@ -168,6 +169,9 @@ def test_annotate_vcf_input_validation(vcf_annotator):
 @pytest.mark.vcr
 def test_get_vrs_object_invalid_input(vcf_annotator, caplog):
     """Test that _get_vrs_object method works as expected with invalid input"""
+    # some tests are check
```

Comment:
Don't think this is complete?

Reply:
Yeah not sure what happened there

Reply:
The author was assassinated mid-sentence